deeposlandia/datasets/__init__.py

```diff
@@ -479,8 +479,14 @@ def populate(
                         ],
                     )
         else:
+            if nb_image_files < nb_images:
+                logger.warning(
+                    "Asking to preprocess %s images, but only got %s files",
+                    nb_images, nb_image_files)
+                nb_images = nb_image_files
+                logger.warning("Preprocessing %s images..", nb_images)
```

Comment:
I guess we're going to preprocess `nb_image_files` images there, aren't we? I'm not sure the logged variable is the good one.

Reply:
Ok, forget my comment, one does `nb_images = nb_iimage_files` one line before...